src/main/java/me/karakelley/tictactoe/GameLoop.java

```diff
@@ -31,10 +31,11 @@ private void play(BoardState boardState, Game game) {
   }
 
   private void nextPlayersTurn(BoardState boardState) {
+    UI.clearScreen();
```

Comment:
`UI` is an instance variable, right? UpperCamelCase is a poor naming convention. I would assume those are static methods on a class if I were glancing at this code.

Reply:
Ok - I will work those into a refactoring PR, thanks!